Source/Diagnostics/MultiDiagnostics.cpp

```diff
@@ -82,11 +86,11 @@ MultiDiagnostics::FilterComputePackFlush (int step, bool force_flush, bool BackT
     int i = 0;
     for (auto& diag : alldiags){
         if (BackTransform) {
-            if (diags_types[i] == DiagTypes::BackTransformed) {
+            if ( (diags_types[i] == DiagTypes::BackTransformed) || (diags_types[i] == DiagTypes::RecordingPlane)) {
```

Comment:
The logic here is hard to follow. At least the name `BackTransfor` is misleading.